src/sentry/integrations/gitlab/integration.py

```diff
@@ -92,6 +92,25 @@ def get_repositories(self, query=None):
         resp = self.get_client().search_group_projects(group, query)
         return [{"identifier": repo["id"], "name": repo["name_with_namespace"]} for repo in resp]
 
+    def get_stacktrace_link(self, repo, filepath, default_version):
+        project_id = repo.config["project_id"]
+        try:
+            # repos are projects in GL so the project_id is the repo id which
+            # GL's API uses instead of slugs like GH
+            self.get_client().check_file(project_id, filepath, default_version)
+        except ApiError as e:
+            if e.code != 404:
```

Comment:
@MeredithAnya I'm not sure this works for Gitlab. I think Gitlab gives us a `400` status code if the file doesn't exist.

Reply:
@scefali when I tested in postman, I kept getting a 404 if it didn't exist

Reply:
@MeredithAnya odd, I remember I kept getting 400s. I'll take your word for it though.